container/memory_store.go

```diff
@@ -4,60 +4,60 @@ import (
 	"sync"
 )
 
-// memoryStore implements a Store in memory.
-type memoryStore struct {
-	s map[string]*Container
-	sync.RWMutex
+// MemoryStore implements a Store in memory.
+type MemoryStore struct {
```

Comment:
Was considering if we should rename this to `Store` or keep it as `MemoryStore`; happy to hear if people have a preference.